basicview.cpp

```diff
@@ -18,38 +18,77 @@ limitations under the License.
 
 #include <QVBoxLayout>
 
+#include <QFontDatabase>
+#include <QFont>
+#include <QDebug>
+#include <QChar>
+
+const QString startIcon = "\uf04b";
+const QString stopIcon = "\uf04d";
+const QString pauseIcon = "\uf04c";
+const QString deleteIcon = "\uf1f8";
+const QString reloadIcon = "\uf021";
+
+void BasicView::setFA(QWidget *wid)
+    {
+    if (QFontDatabase::addApplicationFont(":/images/FontAwesome.otf") < 0)
```

Comment:
I don't think this should be under the images dir

Reply:
how about we change the name to assets and put everything there?